application/src/main/java/uk/nhs/hee/tis/revalidation/messages/payloads/IndexSyncMessage.java

```diff
@@ -22,17 +22,20 @@
 package uk.nhs.hee.tis.revalidation.messages.payloads;
 
 import io.swagger.annotations.ApiModel;
+import java.util.List;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import uk.nhs.hee.tis.revalidation.entity.RevalidationSummary;
 
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
 @ApiModel(description = "Message wrapper for index sync messages")
-public class IndexSyncMessage<T> {
-  private T payload;
+public class IndexSyncMessage {
+
+  private List<RevalidationSummary> payload;
```

Comment:
Had to remove generics due to Java's limitations, I'm not sure we'll be reusing this class much anyway, but let me know if this is not the case